packages/core/src/components/TermList/Term.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import Row from '../Row';
+import Spacing from '../Spacing';
+import Text from '../Text';
+
+export default function Term({
+  label,
+  after = '',
+  endAlign,
+  small = true,
```

Comment:
fudge, one last thing. usually props like this are false by default. can you update it to just `small,` - let it be falsely by default?

Reply:
or have it be small by default, but rename this to be `regular`, and when true, the text small below is false

`Text inline small={!regular}`

Reply:
I think Term label text is almost always small, doing default falsely was the rationale behind callingi this prop `keyTextRegular` originally, but regardless of implementation I think the default behavior should be small text

Reply:
sounds good! so maybe instead of a small=true, a regular=true to override the default of small

Reply:
got it!